android/build.gradle

```diff
@@ -9,7 +9,6 @@ buildscript {
     ext.mockk_version = '1.12.3'
     ext.gradle_maven_publish = '0.22.0'
     ext.purchases_version = '6.0.0-beta.2'
-    ext.billing_unityiap_purchases_version = '3.0.3'
```

Comment:
Hmm we were using BC3 for tests in PHC... It probably works since we don't really run any test that requires it, we just need it to make it compile. Once we bring this back, we probably should update this version though